### ISSUE TITLE
522 when downloading spigot jar

### ISSUE THREAD

**Issue by nhavens** [itzg/docker-minecraft-server#3200]
I've attempted to install Minecraft using the wonderful [itzg minecraft helm chart](https://github.com/itzg/minecraft-server-charts/tree/master/charts/minecraft). My eventual goal is to use world data from a Spigot Minecraft server that ran outside of kubernetes. I have the following in my values.yaml:

```yaml
minecraftServer:
  eula: "TRUE"
  type: "SPIGOT"
```

Unfortunately, my Minecraft pod has been in CrashLoopBackOff status for > 12 hours. In pod logs, I see that it's failing to download the spigot jar.
```
[init] Running as uid=1000 gid=3000 with /data as 'drwxrwsr-x 5 1000 2000 4096 Dec 22 15:37 /data'
[init] Resolving type given SPIGOT
[init] Downloading Spigot from https://download.getbukkit.org/spigot/spigot-1.21.jar ...
curl: (22) The requested URL returned error: 522

ERROR: failed to download from https://download.getbukkit.org/spigot/spigot-1.21.jar
       Visit https://getbukkit.org/download/spigot to lookup the
       exact version or see if download site is unavailable.
       Click into the version entry to find the **exact** version.

rm: cannot remove 'spigot_server-1.21.jar.1': No such file or directory
```

Is this a common occurrence for the default Spigot download source? Can anyone suggest an alternate download source that's more reliable?

**Comment by nhavens** [comment]
Thanks for the speedy response! Sorry about posting the issue against the wrong repo 😊 Thanks for moving it to the right place.

We're up and running with Paper. I'm not sure whether it'd be better to just close this issue or to request that the docs be updated to list Paper a recommended alernative to Spigot. Alternatively, in the [itzg minecraft helm chart](https://github.com/itzg/minecraft-server-charts/tree/master/charts/minecraft), you could do the following to get `type: "SPIGOT"` working:
- set the env var `BUILD_FROM_SOURCE` to `"true"`
- enable the startupProbe (to allow time for compilation)
- set the image tag to `java21-jdk`

**Comment by itzg** [comment]
Yeah, I might at least add an info box at the top of the Spigot/Bukkit page recommending Paper instead.
